index.html

```diff
@@ -0,0 +1,198 @@
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">
+                Graphic Design Course
+              </p>
+              <p class="program-description text-white">
+                Adobe Creative Cloud (Photoshop, Illustrator, XD), Sketch,
+                Figma, Affinity Designer, GIMP
+              </p>
+            </div>
+            <div
+              class="block my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2"
+            >
+              <div class="">
+                <img class="icon" src="icons/exhibition.svg" alt="" />
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">
+                interial Design Course
+              </p>
+              <p class="program-description text-white">
+                Adobe Creative Cloud (Photoshop, Illustrator, XD), Sketch,
+                Figma, Affinity Designer, GIMP
+              </p>
+            </div>
+
+            <div
+              class="block my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2"
+            >
+              <div class="">
+                <img class="icon" src="icons/workshop.svg" alt="" />
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">
+                Architecture Design Tools
+              </p>
+              <p class="program-description text-white">
+                Adobe Creative Cloud (Photoshop, Illustrator, XD), Sketch,
+                Figma, Affinity Designer, GIMP
+              </p>
+            </div>
+            <div
+              class="block blocke my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2"
+            >
+              <div class="">
+                <img class="icon" src="icons/forum.svg" alt="" />
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">Forum to Discuss</p>
+              <p class="program-description text-white">
+                Adobe Creative Cloud (Photoshop, Illustrator, XD), Sketch,
+                Figma, Affinity Designer, GIMP
+              </p>
+            </div>
+
+            <div
+              class="block my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2"
+            >
+              <div class="">
+                <img class="icon" src="icons/ignite.svg" alt="" />
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">
+                Architecture Editors
+              </p>
+              <p class="program-description text-white">
+                Adobe Creative Cloud (Photoshop, Illustrator, XD), Sketch,
+                Figma, Affinity Designer, GIMP
+              </p>
+            </div>
+          </div>
+
+          <button class="join p-4 mt-5">Join CES Global Summit 2020</button>
+          <p class="desktop-buttone">
+            <a href="" class="desktop-button">SEE THE WHOLE PROGRAM</a>
+          </p>
+        </div>
```

Comment:
- [ ] Please change the design of this button on the desktop view to match the given template.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/227248774-d40cb0c7-5ca8-4ea2-b319-b37e5c4ac7ef.png) | ![image](https://user-images.githubusercontent.com/62290545/227248878-5e8cac83-c191-49df-8075-afd28f3ec0e8.png) |

Reply:
Done